### DISCUSSION TITLE
Can I Use a Self Fine-Tuned Whisper Model from the Hugging Face Interface with Faster-Whisper?

### DISCUSSION THREAD

**Discussion by Sun-SunQian** [SYSTRAN/faster-whisper#332] [Q&A]
Hi Faster-Whisper Community,

I'm interested in using Faster-Whisper for my project, with a model that's been fine-tuned to meet my specific requirements.

I've been following the guide on this post to fine-tune my model: https://huggingface.co/blog/fine-tune-whisper. As a result, I've loaded the model from the Hugging Face interface, instead of from the Whisper GitHub repository.

Could anyone provide some guidance on how to incorporate my locally fine-tuned model into Faster-Whisper?

Thanks in advance for your help!

**Comment by phineas-pta** [comment]
u should read README more thoroughly
